core_api/src/modules/user/user.resolver.ts

```diff
@@ -0,0 +1,55 @@
+import { Resolver, Mutation, Arg } from 'type-graphql';
+import { User, Role, Department, Gender } from '../entities.index';
+import * as dotenv from 'dotenv';
+
+dotenv.config();
+
+@Resolver(User)
+export class UserResolver {
+  @Mutation(() => User)
+  async addUser(
+    @Arg('firstname') firstname: string,
+    @Arg('lastname') lastname: string,
+    @Arg('email') email: string,
+    @Arg('roleLabel') roleLabel: string,
+    @Arg('departmentLabel', { nullable: true }) departmentLabel: string,
+    @Arg('genderLabel', { nullable: true }) genderLabel: string
+  ): Promise<User> {
+    const hashedPassword = process.env.TEST_USER_PASSWORD;
```

Comment:
Need to import `hashPassword()` from `/src/utils/auth.utils.ts` and pass TEST_USER_PASSWORD to that in order to get an actual Argon2 hashed string!

Reply:
waiting for your pr to be merged and i'll steal it from you

Reply:
in fact i didn't wait